Update the API "login" action documentation (#286)
kuzzleio/documentation: src/protocols/1/native-protocols/websocket/index.md

---
layout: full.html.hbs
title: WebSocket
order: 0
---
# WebSocket

## Configuration

> The protocol can be configured via [Kuzzle'rc configuration]({{ site_base_path }}guide/1/essentials/configuration/), under ``server > protocols > websocket`` section.

Author: ```suggestion
The protocol can be configured via the [kuzzlerc configuration file]({{ site_base_path }}guide/1/essentials/configuration/), under the ``server > protocols > websocket`` section.
```

| Option | Type | Description | Default |
|---|---|---|---|

> | ``enabled`` | <pre>boolean</pre> | Enables/Disables WebSocket protocol support | ``true`` |

Author: ```suggestion
| ``enabled`` | <pre>boolean</pre> | Enable/Disable WebSocket protocol support | ``true`` |
```

> | ``enabled`` | <pre>boolean</pre> | Enable/Disable WebSocket protocol support | ``true`` |

Reviewer: Missing `heartbeat` documentation (https://github.com/kuzzleio/kuzzle/pull/1289)

Reviewer 2: :+1:

### Configure listening port

<div class="alert alert-warning">

> HTTP, WebSocket and Socket IO protocols share the same underlying server instance. Modifying the listening port will impact all these three protocols.

Author: ```suggestion
HTTP, WebSocket and Socket.IO protocols share the same underlying server instance. Modifying the listening port will impact all these three protocols.
```
